start.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+
+# Add nagios dirs
+mkdir -p /var/log/nagios/archives \
+      /var/log/nagios/rw \
+      /var/log/nagios/spool/checkresults
+
+# Add nagios user
+htpasswd -c -b -s /etc/nagios/passwd ${NAGIOS_USER} ${NAGIOS_PASSWORD}
+
+# Generate command config
+python /opt/rhmap/make-nagios-commands-cfg.py
```

Comment:
The python file should be executable and has a shebang, so no need to call the interpreter here:

```
/opt/rhmap/make-nagios-commands-cfg.py
```

Same below.